lib/iterator.js

```diff
@@ -411,6 +415,24 @@ class EnumerateIterator extends Iterator {
   }
 }
 
+class SkipWhileIterator extends Iterator {
+  constructor(base, predicate, thisArg) {
+    super(base);
+    this.predicate = predicate;
+    this.thisArg = thisArg;
+  }
+
+  next() {
+    let next = this.base.next();
+    if (next.done || !this.predicate) return next;
+    while (!next.done && this.predicate.call(this.thisArg, next.value)) {
+      next = this.base.next();
+    }
+    this.predicate = null;
```

Comment:
Can you add a separate property for this instead of rewriting the existing one, as you did in `Iterator#takeWhile()` [implementation](https://github.com/metarhia/common/pull/188)?